addons/website_forum/models/forum.py

```diff
@@ -1036,10 +1036,10 @@ class Tags(models.Model):
         ('name_uniq', 'unique (name, forum_id)', "Tag name already exists !"),
     ]
 
-    @api.depends("post_ids.tag_ids", "post_ids.state")
+    @api.depends("post_ids", "post_ids.tag_ids", "post_ids.state", "post_ids.active")
```

Comment:
Is 'active' necessary ? It should be taken into account ?

Reply:
That being said, why is it stored ? Can't we unstore it in master and use a read_group, now that it has been significantly improved ?

Reply:
Honestly, it's helping me to compute tags statistics on a forum in #123966 with a simple request on tags instead of posts there.

Reply:
I tend to agree, that it should not be stored (and now that we do a SQL query it shouldn't be that complicated to compute it in SQL, maybe to discuss on the master task)